DigitalLearningSolutions.Data/Services/CourseContentService.cs

```diff
@@ -23,21 +24,52 @@ public CourseContentService(IDbConnection connection, ILogger<CourseContentServi
             this.logger = logger;
         }
 
-        public CourseContent? GetCourseContent(int customisationId)
+        public CourseContent? GetCourseContent(int candidateId, int customisationId)
         {
-            return connection.QueryFirstOrDefault<CourseContent>(
-                @"SELECT Customisations.CustomisationID AS id,
+            CourseContent? courseContent = null;
+            return connection.Query<CourseContent, CourseSection, CourseContent>(
+                @"SELECT TOP 10000 Customisations.CustomisationID AS id,

```

Comment:
Should `TOP 1000` be here? 🙂

Reply:
Nope, sorry about that! Fixed in 0635e51 🙂